### ISSUE TITLE
How do I enable chatbot failback?

### ISSUE THREAD

**Issue by TodaysITSolutions** [forslund/fallback-aiml#2]
Team,

            Thank you so much in advance for working this project, I am very happy to learn that Mycroft has an AIML skill/fallback but I am having trouble with running it. Here are my steps & errors

Platform: Ubuntu server 17.10

Step 1: Install Mycroft in /home - ~/mycroft-core using git clone method

Step 2: Install skill in skills directory & add my own AIML files

Step 3: launch & attempt to start chatbot - no idea how to start the chatbot.

**Comment by ShadowEO** [comment]
I agree, this doesn't seem to work on my mycroft installation either (unless it falls back to AIML only if there's no Internet connection). I assume it was meant to provide conversational responses in the case that a command doesn't match, but all I get is "Sorry, I didn't catch that. Please rephrase and try again."

**Comment by forslund** [comment]
Hi,

this was disabled (in a hard way) back in August when we had some memory issues associated with this skill. I've been meaning to re-enable it now when we've gotten that issue resolved but I've not gotten around to it.

I'll make an update to enable this in the skill settings on home.mycroft.ai during the week.

**Comment by nielstron** [comment]
My pull request #4  includes a settingsmeta.json.ignore that could be used to control the usage of an aiml fallback. Have disabled it atm because if the user does not wish to use aiml, he may uninstall the skill...

**Comment by nielstron** [comment]
#7 will fix this

**Comment by Kareema0** [comment]
I'm trying to use Mycroft git/dev with fallback-aiml/master and "Enable chatbot fallback" set via skill configuration on Mycroft Home. The fallback doesn't seem to work. Is this a known issue? As I'm new to Mycroft - could you give me a hint, how to debug the problem? I didn't see anything wrong in /var/log/mycroft/skills.log so far.

**Comment by th3r00t** [comment]
I am also looking to get this skill working, Any input from the author would be appreciated

**Comment by forslund** [comment]
Hi @th3r00t sorry for the delay in answering. There was an issue due to a change on how boolean settings were reported which made this skill never activate. This was resolved in #19, the update is still pending in mycroft-skills repo so it won't be upgraded unless you ask mycroft to install the beta version.
